common/common.scss

```diff
@@ -120,51 +118,66 @@
         color: var(--primary-medium);
         white-space: nowrap;
         text-overflow: ellipsis;
-        transition: all .25s;
+        transition: all 0.25s;
       }
 
       .d-icon {
         font-size: var(--font-down-1);
         top: 1px;
-      } 
+      }
 
       &:hover {
-        h2 {color: var(--primary);}
+        h2 {
+          color: var(--primary);
+        }
       }
 
       &:after {
         content: "";
         position: relative;
         margin-left: 1em;
-        top: -.3em;
+        top: -0.3em;
         width: 100%;
         border-bottom: 1px solid var(--primary-low);
       }
-   }
+    }
 
     .category-box {
       border-radius: 8px;
       overflow: hidden;
       border: 1px solid;
       border-bottom: 8px solid;
-      transition: all .25s;
+      transition: all 0.25s;
       .category-box-inner {
         border: none;
         padding: 2em;
         @media screen and (max-width: 700px) {
           padding: 2em 1em 1.25em;
-
         }
-        transition: background .25s ease-in-out;
+        transition: background 0.25s ease-in-out;
       }
       .category-box-heading {
-        margin-bottom: .75em;
+        margin-bottom: 0.75em;
       }
-      
+
       &:hover {
-        box-shadow: 0 8px 15px rgba(0,0,0,.1);
+        box-shadow: 0 8px 15px rgba(0, 0, 0, 0.1);
         transform: translate(0, -4px);
       }
     }
   }
-}
\ No newline at end of file
+}
+
+// hides the default categories layout
+
+.navigation-categories .contents .category-boxes {
+  display: none;
+}
+
+@if $show_on_mobile == "true" {
+  .mobile-view {
+    .category-list.with-topics {
+      display: none;
```

Comment:
I was a bit surprised to see: `show_on_mobile == "true" → display: none`. I expected it to be displayed under that condition. Is this correct? 🤔